src/utils/rocm.cpp

```diff
@@ -0,0 +1,226 @@
+#include "hip/hip_runtime.h"
```

Comment:
OCD: Let's move this after the boilerplate.

Reply:
hmm that's odd - I think the hipify script might have added this here 👎

Reply:
lol